webfront/views/cache.py

```diff
@@ -6,11 +6,66 @@
 
 from rest_framework import status
 from webfront.response import Response
+from webfront.views.utils import endpoints
+
 from django.conf import settings
 
 multiple_slashes = re.compile("/+")
 
 
+FIVE_DAYS = 5 * 24 * 60 * 60
+SHOULD_NO_CACHE = -1
+
+names = [ep["name"].lower() for ep in endpoints]
+
+
+def get_timeout_from_path(path, endpoint_levels):
+    parsed = urlparse(path)
+    # process query
+    query = parse_qs(parsed.query, keep_blank_values=True)
+
+    if (  # is requesting by accession
+        len(endpoint_levels) == 3
+        and len([ep for ep in endpoint_levels if ep in names]) == 1
```

Comment:
Can you convert `ep` to lower case? i.e. `len([ep for ep in endpoint_levels if ep.lower() in names]) == 1`